### PR TITLE
nginx extra configs

### PR DESCRIPTION
**Pull request by JesseGoodier**
## What does this PR change?

adds helm template modifying nginx config to allow for adjusting buffer size to accommodate OIDC and SAML

## Does this PR rely on any other PRs?

No

## How does this PR impact users? (This is the kind of thing that goes in release notes!)

Added helm template modifying nginx config to allow for adjusting buffer size to accommodate OIDC and SAML

## Links to Issues or ZD tickets this PR addresses or fixes

- https://github.com/kubecost/cost-analyzer-helm-chart/issues/2393

## How was this PR tested?

local helm install, verify nginx starts and forwards traffic

## Have you made an update to documentation?

TBD, but there is a note in the values file

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -319,16 +319,23 @@ kubecostFrontend:
     requests:
       cpu: "10m"
       memory: "55Mi"
-    #limits:
-    #  cpu: "100m"
-    #  memory: "256Mi"
+    # limits:
```
Comment (jessegoodier) [inline cost-analyzer/values.yaml:322]: Just making spacing consistent.

**Comment by jessegoodier** [comment]
related to https://github.com/kubecost/cost-analyzer-helm-chart/issues/2393

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml [hunk 2]
```diff
@@ -111,9 +111,15 @@ data:
         server_name _;
         root /var/www;
         index index.html;
-        large_client_header_buffers 4 32k;
+
         add_header Cache-Control "must-revalidate";
```
Comment (jessegoodier) [inline cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml:116]: If no value is passed, nginx will have the same config as previous releases.

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml [hunk 3]
```diff
@@ -111,9 +111,15 @@ data:
...
 
+        {{- if.Values.kubecostFrontend.extraServerConfig }}
```
Comment (jessegoodier) [inline cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml:117]: If no value is passed, nginx will have the same config as previous releases.

**Review by nealormsbee — APPROVED** [review]
This looks fine to me

**Pull request merged**